internal/tutils/psql.go

```diff
@@ -87,6 +89,9 @@ func (p *PSQLContainer) execCommand(args ...string) (string, error) {
 	}
 	/* #nosec G204 */
 	out, err := exec.Command(rt, args...).CombinedOutput()
+	if err != nil {
+		err = fmt.Errorf("command %s %s error: %w, output: %s", rt, args, err, out)
+	}
```

Comment:
The PR contains couple of error printing, during initial setup I had problems (missing `podman` or misconfiguration there) and it was hard to troubleshoot without these.

Reply:
would be nice so split out test improvements to a separate commit.